### DISCUSSION TITLE
Is it possible to set mirror  url  for ghcr.io?

### DISCUSSION THREAD

**Discussion by liangyuanpeng** [Homebrew/discussions#4127] [Everyday usage]
### Output of `brew config`

```shell
:)
```

### Output of `brew doctor`

```shell
:)
```

### Description of issue

I want to set mirror for ghcr.io, is it possible? Remind me if i missed something,Thanks any help. 

Like some env :  `HOMEBREW_GHCR_MIRROR` or `HOMEBREW_XXX`

**Comment by alebcay** [comment]
Yes, see the full list of environment variables that can be set at https://docs.brew.sh/Manpage#environment. `HOMEBREW_ARTIFACT_DOMAIN` and `HOMEBREW_BOTTLE_DOMAIN` may be of interest to you; please refer to the linked doc for details on what these do.
